docker/DockerBuild.sh

```diff
@@ -31,12 +31,13 @@ echo "CMake Source Directory: $CMAKE_SOURCE_DIR"
 echo "MiNiFi Package: $MINIFI_SOURCE_CODE"
 
 # Copy the MiNiFi source tree to the Docker working directory before building
+rm -rf $CMAKE_SOURCE_DIR/docker/minificppsource
 mkdir -p $CMAKE_SOURCE_DIR/docker/minificppsource
 rsync -avr \
       --exclude '/*build*' \
       --exclude '/*_repository*' \
       --exclude '/logs' \
-      --exclude '/cmake' \
+      --exclude '/cmake-build-*' \
```

Comment:
@arpadboda  This was added in https://github.com/apache/nifi-minifi-cpp/pull/634 - I am not sure what was the original intent here, but this excludes the whole cmake directory, which messes up the whole build process. I changed it to this, based on the .gitignore, but I want to make sure there was no other consideration behind it that I would break with this.

Reply:
I wanted to avoid uploading CMakeCache and cmake-build-debug to the container, thanks for fixing.